cmd/minibroker/main.go

```diff
@@ -77,12 +86,12 @@ func run() error {
 
 func runWithContext(ctx context.Context) error {
 	if flag.Arg(0) == "version" {
-		fmt.Printf("%s/%s\n", path.Base(os.Args[0]), "0.1.0")
+		klog.V(0).Infof("%s/%s", path.Base(os.Args[0]), "0.1.0")
 		return nil
 	}
 	if (options.TLSCert != "" || options.TLSKey != "") &&
 		(options.TLSCert == "" || options.TLSKey == "") {
-		fmt.Println("To use TLS, both --tlsCert and --tlsKey must be used")
+		klog.V(0).Infoln("To use TLS, both --tlsCert and --tlsKey must be used")
```

Comment:
I'll return an error instead. As for it being level 0, it means it will *always* be printed to the output. The default log level for the library is 2, but I'm switching to 3 instead of the hyper-mega-ultra verbose 5 that we currently use.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/57/commits/3cd42233036a82a9c6fa85de0524352be0192102.

Reply:
Ugh, the documentation (`If the level in the call to V is at least the value of -v… the V call will log`) is straight up wrong; the actual code says `if logging.verbosity.get() >= level { … do logging }` instead.  The comparison goes the opposite way of the documentation.

Reply:
Ah! I just found [guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md) on the logging levels. Of course there's no actual link to that from the logging package…